base/applications/mspaint/lang/ro-RO.rc

```diff
@@ -17,8 +17,8 @@ BEGIN
     BEGIN
         MENUITEM "&Nou\tCtrl+N", IDM_FILENEW
         MENUITEM "&Deschidere...\tCtrl+O", IDM_FILEOPEN
-        MENUITEM "&Păstrează\tCtrl+S", IDM_FILESAVE
-        MENUITEM "Păst&rare în...", IDM_FILESAVEAS
+        MENUITEM "S&alvează\tCtrl+S", IDM_FILESAVE
+        MENUITEM "Sal&vează ca...", IDM_FILESAVEAS
```

Comment:
Accelerator &v collides with
 MENUITEM "Pre&vizionează pagina", IDM_FILEPAGEVIEW
in the same menu.